exec/java-exec/src/main/java/org/apache/drill/exec/server/rest/DrillRestServer.java

```diff
@@ -73,6 +78,11 @@
 import java.util.ArrayList;
 import java.util.List;
 
+@OpenAPIDefinition(info = @Info(title = "Apache Drill REST API", description = "OpenAPI Specification", license = @License(name = "Apache Software Foundation (ASF)", url = "http://www.apache" + ".org/licenses/LICENSE-2.0"), contact = @Contact(name = "Apache Drill", url = "https://drill.apache.org/")))/*,
+tags = {
+  @Tag(name = "My Tag 1", description = "Tag 1's Description", externalDocs = @ExternalDocumentation(description = "docs description1")),
+  @Tag(name = "My Tag 2", description = "Tag 2's Description", externalDocs = @ExternalDocumentation(description = "docs description2"))})*/
```

Comment:
Please remove commented out code, here and elsewhere.   The line above, please break that up a bit so that it fits w/o horizontal scroll.